source/common/local_reply/local_reply.cc

```diff
@@ -23,7 +23,7 @@ class BodyFormatter {
         content_type_(Http::Headers::get().ContentTypeValues.Text) {}
 
   BodyFormatter(const envoy::config::core::v3::SubstitutionFormatString& config,
-                Server::Configuration::CommonFactoryContext& context)
+                Server::Configuration::FactoryContext& context)
```

Comment:
Safe because the `BodyFormatter` only be used in the listener scope.

Reply:
Yeah, generic factory context is enough for this scenario. I use FactoryContext just for `convenience`. I will update it.